### PR TITLE
fix(form): decrease min-height of rows

### PR DESCRIPTION
**Pull request by LucyChyzhova**
fix: https://github.com/Lundalogik/crm-feature/issues/3316
## Review:
- [ ] Commits are [atomic](https://seesparkbox.com/foundry/atomic_commits_with_git)
- [ ] Commits have the correct *type* for the changes made
- [ ] Commits with *breaking changes* are marked as such

### Browsers tested:
(Check any that applies, it's ok to leave boxes unchecked if testing something didn't seem relevant.)

Windows:
- [ ] Chrome
- [ ] Edge
- [ ] Firefox

Linux:
- [ ] Chrome
- [ ] Firefox

macOS:
- [ ] Chrome
- [ ] Firefox
- [ ] Safari

Mobile:
- [ ] Chrome on Android
- [ ] iOS

### CODE REVIEW

File: src/components/form/examples/span-fields-schema.ts [hunk 1]
```diff
@@ -46,7 +46,7 @@ export const schema = {
                 },
                 layout: {
                     colSpan: 2,
-                    rowSpan: 3,
+                    rowSpan: 1,
```
Comment (Kiarokh) [inline src/components/form/examples/span-fields-schema.ts:49]: this shouldn't be a part of the change. 😄

Comment (LucyChyzhova) [reply src/components/form/examples/span-fields-schema.ts:49]: ah, I will remove it, of course 🤗

**Review by Kiarokh — DISMISSED** [review]
awesome!

**Review by Kiarokh — APPROVED** [review]

**Pull request merged**